src/routes/NewBoard/NewBoard.scss

```diff
@@ -33,6 +33,10 @@
     }
   }
 }
+.new-board__mode:last-child {
+  grid-column: 1 / -1;
+  max-width: 100%;
+}
```

Comment:
not sure about this, looks kinda weird as well 👀 should probably ask @SelinaBuff what's better

Reply:
I talked to @SelinaBuff  about this, and the plan was to have the import button across the entire width so that it lines up nicely with the two rows.